node/pkg/accountant/accountant_test.go

```diff
@@ -132,7 +132,7 @@ func newAccountantForTest(
 }
 
 // Converts a string into a go-ethereum Hash object used as test input.
-func hashFromString(str string) ethCommon.Hash {
+func hashFromString(str string) ethCommon.Hash { //nolint:unparam
```

Comment:
Why is this necessary?

Reply:
hashFromString is only ever called with a single static value, which is expected in this test file, but that's why the linter complains.